sandpiper/processes/wps_resolve_rules.py

```diff
@@ -1,13 +1,14 @@
-from pywps import Process, LiteralInput, ComplexInput, ComplexOutput, FORMATS, Format
-from pywps.app.Common import Metadata
 import json
 import os
+from tempfile import NamedTemporaryFile
```

Comment:
Are we using `tempfile` in here?

Reply:
No I will remove and use flake8 to double check all the files, thanks for the reminder.